week-07/FoxClub/day-05/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,48 @@
+﻿@model day05.Models.Fox
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <title>@ViewBag.Title</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+</head>
+<body>
+    @Html.Partial("_navbar", Model.Name)
+    <main>
+        <img src="~/img/greenfox.png" />
```

Comment:
I'd avoid the `~`. I'd go for either of these

1) `/img/greenfox` - absolute URL
2) `./img/greenfox` - relative URL, usually this causes trouble (if route structure changes) but it can be useful in some cases
3) Using HTML.ActionLink to generate the URL - probably the best option as it the least likely to break if something changes in the app.